Cabal/src/Distribution/Simple/Command.hs

```diff
@@ -134,6 +134,14 @@ type ArgPlaceHolder = String
 
 -- | Create an option taking a single OptDescr.
 --   No explicit Name is given for the Option, the name is the first LFlag given.
+--
+-- Example: @'option' sf lf d get set@
+-- * @sf@: Short option name, for example: @['d']@. No hyphen permitted.
+-- * @lf@: Long option name, for example: @["debug"]@. No hyphens permitted.
+-- * @d@: Description of the option, shown to the user in help messages.
+-- * @get@: Get the current value of the flag.
+-- * @set@: Set the value of the flag. Gets the current value of the flag as a
+--          parameter.
```

Comment:
BTW, I am choosing this style because HLS has a good time displaying these docs to me while hovering.

Reply:
could you share a screenshot of what it looks like to you? :)

Reply:
Always:
![image](https://user-images.githubusercontent.com/8463814/212863290-2a16d455-e6cb-4002-a93d-2b3d119aa133.png)

Reply:
Fixed the typo in `[ d ]` to `['d']`, but haven't updated the screenshot

Reply:
lovely, thank you!